src/command-line/users/list.js

```diff
@@ -25,16 +18,17 @@ program
 			return;
 		}
 
-		if (users.length > 0) {
-			log.info("Users:");
-			users.forEach((user, i) => {
-				log.info(`${i + 1}. ${colors.bold(user)}`);
-			});
-		} else {
+		if (users.length === 0) {
 			log.info(
 				`There are currently no users. Create one with ${colors.bold(
 					"thelounge add <name>"
 				)}.`
 			);
+			return;
```

Comment:
I'm a fan of return early, keep the happy path to the left.
If you disagree heavily please tell me and I'll adapt my style.

Reply:
I think either way is fine. Code style seems like a fairly low stakes thing in an OSS code case with an auto formatter too.